code-of-conduct-languages/hi.md

```diff
@@ -0,0 +1,30 @@
+## CNCF कम्युनिटी कोड ओफ़ कंडक्ट v1.0
+
+### कंट्रीब्यूटर कोड ओफ़ कंडक्ट
+
+इस प्रॉजेक्ट के कंट्रीब्यूटर और मेंटेनर के रूप में, और एक कम्युनिटी को बढ़ावा देने के हित में, हम उन सभी लोगों का सम्मान करने की प्रतिज्ञा करते हैं जो रिपोर्टिंग issues के माध्यम से योगदान करते हैं, फीचर request पोस्ट करते हैं, प्रलेखन अपडेट करते हैं, pull requset या patches, और अन्य गतिविधियों में भाग लेते हैं।
```

Comment:
Can we do it -> ....जो issues रिपोर्टिंग के माध्यम से योगदान करते हैं...

Reply:
This is a tuff one 🤔

Any comments on this @Rajakavitha1?

Reply:
How about ......जो issues रिपोर्ट  करते हैं... to keep it in parallel with the rest of the tasks such as प्रलेखन अपडेट करते हैं

Reply:
Okay then.

Let's go with `issues रिपोर्टिंग`.

BTW, is it `issue रिपोर्टिंग` or `issues रिपोर्टिंग`

Reply:
I would go for `issues`.